dashboard/config/libraries/GameLabJr.interpreted.js

```diff
@@ -1,3 +1,5 @@
+var extraArgs = null;
```

Comment:
Is there a benefit of declaring the variables here rather than handling in checking touching? ( ex: if(extraArgs){ } else {})

Reply:
The problem is making sure the variable is defined if `getThisSprite` is used *outside* the touching event block. 
The `this/other sprite` block generates code: `getThisSprite('this', extraArgs)`. This works as expected if used in a touching event block, because the touch event block generates code: `checkTouching('when', sprite1, sprite2, function(extraArgs) {...})`. `extraArgs` is defined in the native event handling code and then passed as a parameter to the callback, so everything  works as expected when the `this/other sprite` block is used correctly. However, if that block is used outside the event block, it still generates `getThisSprite('this', extraArgs)`, but now `extraArgs` is an undeclared variable. This PR solves that problem by globally defining `extraArgs` to be null.

Reply:
Ahhhh, I follow. Thanks for the explanation.